aiounifi/interfaces/connectivity.py

```diff
@@ -71,6 +71,12 @@ async def login(self) -> None:
             "rememberMe": True,
         }
 
+        if hasattr(self.config, "site") and self.config.site is not None:
+            auth["site_name"] = self.config.site
+
+        if hasattr(self.config, "for_hotspot") and self.config.for_hotspot is not None:
+            auth["for_hotspot"] = self.config.for_hotspot
+        
```

Comment:
```suggestion
        if self.config.hotspot_manager_account:
            auth["for_hotspot"] = True
            auth["site_name"] = self.config.site
        
```
Allright, lets keep them together and only add them when its for a hotspot manager